iina/FFmpegController.m

```diff
@@ -528,6 +529,7 @@ + (NSImage *)createNSImageWithContentsOfURL:(nonnull NSURL *)url
              pFrame->format);
       case AV_PIX_FMT_ARGB: // WebP with screenshot-webp-lossless mpv option enabled.
       case AV_PIX_FMT_RGB24: // JPEG XL SDR video.
+      case AV_PIX_FMT_RGBA64LE: // JPEG XL SDR video.
```

Comment:
Hmm, maybe I am not understanding the scope of these changes. Why is this here?

Reply:
The Issue #5212 originated from issue #5200 which was about properly handling a video with BT.2020 primaries and a SDR gamma. A JPEG XL screenshot of that kind of video uses this pixel format and before this change was triggering the "Unexpected pixel format" warning.